paddle/fluid/operators/activation_op.h

```diff
@@ -301,8 +303,22 @@ template <typename T>
 struct GeluFunctor : public BaseActivationFunctor<T> {
   template <typename Device, typename X, typename Out>
   void operator()(Device d, X x, Out out) const {
+#if defined(PADDLE_WITH_MKLML) && !defined(_WIN32) && !defined(__APPLE__) && \
```

Comment:
Please try to do not add `#ifdef` here.

Can GetBlas should be common method, you can refer to `GetBlas::VMUL`

Reply:
Because GetBlas function need the extra context parameters that it be not exist in activition_op.h, so now it keep current status.

auto blas = math::GetBlas<platform::CPUDeviceContext, T>(**ctx**);